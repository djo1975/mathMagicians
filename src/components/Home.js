import React from 'react';
import Header from './Header';

const Home = () => (
  <div>
    <Header />
    <h2>Welcome to our page!</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas tincidunt velit vitae
      justo lobortis, ut tristique magna porttitor. Etiam malesuada eu mauris vel rhoncus. Sed
      rutrum aliquet orci, ac bibendum mauris luctus ut. Nulla a dolor non sapien faucibus pulvinar.
      In scelerisque odio vel metus pellentesque, ut eleifend nisi bibendum.
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas tincidunt velit vitae
      justo lobortis, ut tristique magna porttitor. Etiam malesuada eu mauris vel rhoncus.
      Sed rutrum aliquet orci, ac bibendum mauris luctus ut. Nulla a dolor non sapien faucibus
      pulvinar. In scelerisque odio vel metus pellentesque, ut eleifend nisi bibendum.
    </p>
  </div>
);

export default Home;
