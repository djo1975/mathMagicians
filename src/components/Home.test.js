import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import Home from './Home';

Enzyme.configure({ adapter: new Adapter() });

describe('Home component', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
