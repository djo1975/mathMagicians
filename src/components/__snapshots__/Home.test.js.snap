// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly 1`] = `ShallowWrapper {}`;
